cloud_dashboard/src/App.tsx

```diff
@@ -22,7 +22,11 @@ import 'leaflet/dist/leaflet.css';
 import "react-loader-spinner/dist/loader/css/react-spinner-loader.css";
```

Comment:
@RyoYamashita765 This can be:

```suggestion
import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css';
```

Reply:
@naoi After examining the source code again, it was found that the quotation marks in the import statement were mixed up. I will fix it later.